pkg/monitoring/alloy/testdata/alloy_config_defaulttenant.mc.river

```diff
@@ -93,6 +108,11 @@ prometheus.operator.podmonitors "giantswarm" {
       values   = ["giantswarm"]
     }
   }
+  rule {
+    action = "replace"
+    target_label = "team"
+    source_labels = ["__meta_kubernetes_pod_label_application_giantswarm_io_team"]
```

Comment:
I'm not sure we should use this for non legacy stuff. We probably should use a validé upstream label instead: thinking

Reply:
For now that's the label we have everywhere, I don't want to make thing more complicated. There is also a fair amount of app/services which do not have this label yet. My thinking is:
- First get this new label on metrics
- Make `JobScrapingFailure` a page alert for atlas only, and adjust it
- Open `JobScrapingFailure` to all team
- Push team to add the `application.giantswarm.io/team` label on their apps <= this might be a time where we also decide to change the label